app/src/main/java/com/fstyle/structure_android/screen/main/MainActivity.java

```diff
@@ -39,32 +47,29 @@
 
     private DialogManager mDialogManager;
 
-    @Validation({
-            @Rule(types = {
-                    ValidType.NG_WORD, ValidType.NON_EMPTY
-            }, message = R.string.error_unusable_characters)
-    })
+    @Validation({@Rule(types = {ValidType.NG_WORD, ValidType.NON_EMPTY}, message = R.string
+            .error_unusable_characters)})
     private String mKeyWord;
-    @Validation({
-            @Rule(types = ValidType.VALUE_RANGE_0_100, message = R.string
-                    .error_lenght_from_0_to_100),
-            @Rule(types = ValidType.NON_EMPTY, message = R.string.must_not_empty)
-    })
+    @Validation({@Rule(types = ValidType.VALUE_RANGE_0_100, message = R.string
+            .error_lenght_from_0_to_100), @Rule(types = ValidType.NON_EMPTY, message = R.string
+            .must_not_empty)})
     private int mLimit;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        UserRepository userRepository =
-                new UserRepository(null, new UserRemoteDataSource(NameServiceClient.getInstance()));
+        ((MainApplication) getApplication()).getAppComponent().inject(this);
+
+        UserRepository userRepository = new UserRepository(null, new UserRemoteDataSource
+                (mNameApi));
```

Comment:
ERROR: 'operator new lparen' have incorrect indentation level 16, expected level should be 8.